<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CacheConfig</class>
 <widget class="QWidget" name="CacheConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" rowspan="2" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Tag Bits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="tagBits"/>
       </item>
      </layout>
     </item>
     <item row="1" column="3" rowspan="2" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Line Bits</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="indexBits"/>
       </item>
      </layout>
     </item>
     <item row="4" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Replacement Policy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="replacementCombo"/>
       </item>
      </layout>
     </item>
     <item row="0" column="1" colspan="4">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cache Configuration</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="3" colspan="2">
      <widget class="QPushButton" name="updateButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Update Configuration</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Associativity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="associativityNum"/>
       </item>
      </layout>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="byteFieldBits">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
